params/config.go

```diff
@@ -925,3 +925,22 @@ func NetworkIDByChainName(chain string) uint64 {
 		return config.ChainID.Uint64()
 	}
 }
+
+func Eth2DeployBlockNumber(c *ChainConfig) uint64 {
```

Comment:
Maybe calc this value once when app start and adjust cfg.prune.Receipts according to this numbers?